model/glove/glove.go

```diff
@@ -0,0 +1,257 @@
+// Copyright © 2017 Makoto Ito
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package glove
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"math"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/chewxy/lingo/corpus"
+	"github.com/pkg/errors"
+
+	"github.com/ynqa/word-embedding/model"
+)
+
+// GloVe stores the configs for GloVe models.
+type GloVe struct {
```

Comment:
While GloVe is the proper name (Global Vector) I think it's quite terrible to have random Uppercase letters in the middle of a name that is not a word. Perhaps stick with `Glove` ? Just an opinion

Reply:
Nope, I'll rename it, by the way, why is it terrible there are uppercases in the middle?